package.json

```diff
@@ -15,16 +15,13 @@
     "storybook": "start-storybook -p 9009 --ci",
     "dev": "yarn storybook",
     "build-storybook": "build-storybook",
-    "// build": "For context on tsconfig replacements in build scripts, see https://github.com/radix-ui/primitives/pull/361#discussion_r555004944",
-    "build": "yarn build:config && yarn build:packages && yarn build:cleanup",
-    "build:config": "mv tsconfig.json tsconfig.tmp.json && mv tsconfig.production.json tsconfig.json",
-    "build:packages": "parcel build 'packages/*/*/' --no-cache",
-    "build:cleanup": "mv tsconfig.json tsconfig.production.json && mv tsconfig.tmp.json tsconfig.json",
+    "build": "preconstruct build",
     "publish": "yarn bump && yarn build && yarn workspaces foreach -pv --exclude primitives --exclude ssr-testing npm publish --tolerate-republish --access public",
     "clean": "yarn workspaces foreach -pv --exclude primitives --exclude ssr-testing run clean",
     "reset": "yarn clean && rm -rf node_modules .yarn/cache .parcel-cache",
     "bump": "yarn version apply --all",
-    "bump:check": "yarn version check"
+    "bump:check": "yarn version check",
+    "_postinstall": "preconstruct dev"
```

Comment:
The current version of Yarn fails executing this `postinstall` with:
```
➤ YN0006: │ primitives@workspace:. lists build scripts, but is referenced through a soft link. Soft links don't support build scripts, so they'll be ignored.
➤ YN0007: │ primitives@workspace:. must be built because it never did before or the last one failed
➤ YN0009: │ primitives@workspace:. couldn't be built successfully (exit code 1, logs can be found here: /private/var/folders/ws/tlwxqvs5407f21nxy7w8bgr80000gp/T/xfs-e80184bd/build.log)
➤ YN0009: │ primitives@workspace:. couldn't be built successfully (exit code 1, logs can be found here: /private/var/folders/ws/tlwxqvs5407f21nxy7w8bgr80000gp/T/xfs-e80184bd/build.log)
```

From what I recall - at least this error message is a bug and it has already been fixed in Yarn so you could try switching to the version from their master/main branch.